MalApi/MyAnimeListApi.cs

```diff
@@ -16,7 +15,14 @@ namespace MalApi
     /// </summary>
     public class MyAnimeListApi : IMyAnimeListApi
     {
-        private const string MalAppInfoUri = "https://myanimelist.net/malappinfo.php?status=all&type=anime";
+        private static readonly string AnimeDetailsUrlFormat = "https://myanimelist.net/anime/{0}";
+
+        private const string MalAppAnimeInfoUri = "https://myanimelist.net/malappinfo.php?status=all&type=anime&u={0}";
+        private const string MalAppMangaInfoUri = "https://myanimelist.net/malappinfo.php?status=all&type=manga&u={0}";
+
+        private const string MalAppAnimeUpdateUri = "https://myanimelist.net/api/animelist/update/{0}.xml";
```

Comment:
No need for "App" in the name since it has nothing to do with malappinfo.php. Also, this is a format string rather than a string that can be used directly, but the name does not indicate this. Consider moving the format string directly into the function that uses it so that there is no distance between the format string and the string.Format call. That way it is clear where the format arguments get put and how many are needed.